cpp/src/strings/split/split.cu

```diff
@@ -506,8 +506,8 @@ std::unique_ptr<table> split_fn(strings_column_view const& strings_column,
   }
 
   // create working area to hold all token positions
-  rmm::device_vector<string_index_pair> tokens(columns_count * strings_count);
-  string_index_pair* d_tokens = tokens.data().get();
+  rmm::device_uvector<string_index_pair> tokens(columns_count * strings_count, stream);
+  string_index_pair* d_tokens = tokens.data();
```

Comment:
I'm making more involved changes to this file in #7427

Reply:
Resolved (there weren't conflicts in this file).